src/loaders/JsonImageInfoLoader.ts

```diff
@@ -287,16 +279,10 @@ class JsonImageInfoLoader implements IVolumeLoader {
         // done with img, iData, and canvas now.
 
         for (let ch = 0; ch < Math.min(batch.length, 4); ++ch) {
-          volume.setChannelDataFromAtlas(batch[ch], channelsBits[ch], w, h);
-
-          const cacheInsertDims = {
-            scale: 0,
-            time: volume.loadSpec.time,
-            channel: batch[ch],
-          };
-          cacheStore && cache?.insert(cacheStore, volume.channels[batch[ch]].volumeData.buffer, cacheInsertDims);
-
-          onChannelLoaded?.(volume, batch[ch]);
+          const chindex = batch[ch];
```

Comment:
Nit: Maybe spell out `channelIndex`, or use capitalization for `chIndex`?

Reply:
This is matching a naming convention above, around line 218